command/genesis/genesis.go

```diff
@@ -207,7 +207,7 @@ func setFlags(cmd *cobra.Command) {
 // with running partners
 func setLegacyFlags(cmd *cobra.Command) {
 	// Legacy chainid flag
-	cmd.Flags().Uint64Var(
+	cmd.Flags().Int64Var(
```

Comment:
I just thought chain ID could be unsigned int

Reply:
I thought the same at first, but after our discussions with Vuk, we decided that user should be able to give negative chain id if they feel like it.
But now I realized that if that is the case, `GetChainID` function inside `blockchain_wrapper.go` should also return int64 instead of uint64. It would be weird if user entered a negative number as chain id, but retrieved a very big uint64 number.

Reply:
I think there is no problem even if chain ID is negative but I'm not sure interface using hex value can adopt that.

Reply:
Done